api/response/schema_query.py

```diff
@@ -81,12 +83,15 @@ def get_room_response(self, room_response, room_id):
                 question_type.value
             )(responses.response)
             resolved = responses.resolved
+            archived = responses.state.value
```

Comment:
I believe this variable refers to the state of the response whether archived or not so it should just be named `state`

Reply:
Done